src/components/Notifications/NotificationsList.tsx

```diff
@@ -286,49 +289,44 @@ export default function NotificationsList({
 
   let manageResults: any = null;
 
-  const unsubscribe = () => {
-    navigator.serviceWorker.ready
-      .then(function (reg) {
-        setIsSubscribing(true);
-        reg.pushManager
-          .getSubscription()
-          .then(function (subscription) {
-            subscription
-              ?.unsubscribe()
-              .then(async function (_successful) {
-                const data = {
-                  pf_endpoint: "",
-                  pf_p256dh: "",
-                  pf_auth: "",
-                };
-
-                await request(routes.updateUserPnconfig, {
-                  pathParams: { username: username },
-                  body: data,
-                });
-
-                Warn({
-                  msg: t("unsubscribed_successfully"),
-                });
-
-                setIsSubscribed("NotSubscribed");
-                setIsSubscribing(false);
-              })
-              .catch(function (_e) {
-                Error({
-                  msg: t("unsubscribe_failed"),
-                });
-              });
-          })
-          .catch(function (_e) {
-            Error({ msg: t("subscription_error") });
+  const unsubscribe = async () => {
+    try {
+      const reg = await navigator.serviceWorker.ready;
+
+      if (!reg.pushManager) {
+        Error({ msg: t("unsubscribe_failed") });
+        return;
+      }
+
+      setIsSubscribing(true);
+
+      const subscription = await reg.pushManager.getSubscription();
+
+      if (subscription) {
+        try {
+          await subscription.unsubscribe();
+
+          await request(routes.updateUserPnconfig, {
+            pathParams: { username },
+            body: {
+              pf_endpoint: "",
+              pf_p256dh: "",
+              pf_auth: "",
+            },
           });
-      })
-      .catch(function (_e) {
-        Sentry.captureException(_e);
-      });
-  };
 
+          setIsSubscribed("NotSubscribed");
```

Comment:
Any reason why unsubscribed successfully notification was removed? If not, let's add it back.

Otherwise, LGTM.